src/ekat/physics/base_constants.cpp

```diff
@@ -0,0 +1,114 @@
+/* This file was generated by EKAT with generate_constants.hpp and
```

Comment:
Should this file be source-controlled? Also, given that we decided against the "derived constants" idea that we were kicking around, maybe "base" doesn't mean as much as it used to. What about `physical_constants.xyz` for the generated files?

Reply:
this should not be source-controlled.  I put it in this draft so that you and Luca could see the auto-generated code easily, and so that the tests will run.   But I also put these filenames in the `.gitignore` file; if we decide to go with this approach, I'll `git rm` the auto-gen files, once we make sure that we can test them properly.

Reply:
> What about physical_constants.xyz for the generated files?

Ok with me, I had them named this way to start.